documentation/binding-data/tutorial-flow-data-provider.asciidoc

```diff
@@ -272,28 +272,45 @@ The lazy loaded binding in `ComboBox` is always filtered by the string typed
 in by the end user.
 Initially, when there is no filter input yet, the filter is an empty string.
 
-// TODO change this into data view style once the changes are in ComboBox
-
-`ComboBox` currently uses the old data provider naming convention.
-Also, the count of total matches is currently required.
+*The `ComboBox` examples listed below use an API available only in Vaadin 18+.*
```

Comment:
Maybe point out what the API is, like with
"..examples below use the new data API available since Vaadin 18 where the item count query is not needed for fetching items."
?

Reply:
done